### ISSUE TITLE
Support OSGi Factory Configurations

### ISSUE THREAD

**Issue by absa-piotr-andruszkiewicz** [wttech/aemc#133]
Hello,

I've just wrote an Ansible task to save a configuration:

```yaml
---
- name: Enable logging
  wttech.aem.osgi_config:
    command: save
    pid: org.apache.sling.commons.log.LogManager.factory.config~tarmk.log
    props:
      org.apache.sling.commons.log.level: info
      org.apache.sling.commons.log.file: logs/tarmk.log
      org.apache.sling.commons.log.pattern: "{0,date,dd.MM.yyyy HH:mm:ss.SSS} *{4}* [{2}] {3} {5}"
      org.apache.sling.commons.log.names:
        - org.apache.jackrabbit.oak.segment
```

But it didn't match the factory and ended up like this:
<img width="1022" alt="image" src="https://github.com/wttech/aemc/assets/133335665/6bb323a6-ef8c-491c-b7e2-a5b16ea0f62b">

Is there a way to provide the factory PID? I didn't find an appropriate flag in the CLI help.
Or maybe it could be deduced from the configuration PID?

Cheers!
Piotr

**Comment by krystian-panek-wttech** [comment]
In GAP the case have been covered. Please review the both impls to find the meaningful difference. Then propose a fix. Thx! 😁

**Comment by krystian-panek-wttech** [comment]
Probably tilde should be changed to dot underscore or hyphen or sth

**Comment by piotr-andruszkiewicz-wttech** [comment]
https://github.com/wttech/aemc/pull/140

**Comment by krystian-panek-wttech** [comment]
fixed by #140 ; thanks @piotr-andruszkiewicz-wttech
